app/models/step_by_step_page.rb

```diff
@@ -75,7 +75,7 @@ def mark_as_scheduled
   end
 
   def mark_as_unscheduled
-    update_attribute(:status, "draft")
+    update_attribute(:status, "approved_2i")
```

Comment:
To future-proof this, I'd be tempted to `update_attribute(:status, status_was)`. I can't think of a situation where we'd want to go to anything other than `approved_2i` at the moment, but if/when we implement fact-check, which may or may not be an optional workflow, `approved_2i` may not always be correct. What we're effectively doing is hitting 'Undo' and we want to go back to the previous state.

Reply:
I talked about this with Bruce and we weren't sure how long/well `status_was` persists. We were worried that if the app is restarted this information may be lost.
I see what you mean as I imagined we will have a general `approve` status that would require step by step to be both `approved_2i` and `fact_checked` so we would need to change this to `approved` in the future. 
I will read about this and if it's safe enough we should do it.